src/ray/gcs/gcs_server/test/gcs_server_rpc_test.cc

```diff
@@ -410,6 +410,8 @@ class GcsServerTest : public RedisServiceManagerForTest {
   rpc::GcsNodeInfo GenGcsNodeInfo(const std::string &node_id) {
     rpc::GcsNodeInfo gcs_node_info;
     gcs_node_info.set_node_id(node_id);
+    gcs_node_info.set_node_manager_address("127.0.0.1");
+    gcs_node_info.set_node_manager_port(6000);
```

Comment:
Is this port fake or real? If it's a real port that will be used, we'd better not use a hard-coded one.

Reply:
In fact, this port is only used to build an endpoint. You can fill in any value here, as long as it does not affect the construction of `boost::asio::ip::tcp::endpoint` on the server side.